magento/pwa-studio
# #2797. Re-factor Edit Product to use Dialog

## Description
Added Dialog component to render display Product Edit Form in `packages/venia-ui/lib/components/CartPage/ProductListing/EditModal/productForm.js` component

## Related Issue





Closes #2797.

## Acceptance



### Verification Stakeholders

### Specification



### Verification Steps


1. Add product to Cart
2. Go to Cart Page
3. Expand Product  menu (3 dots on the right of each product)
4. Click "Edit item"
5. Modal with Product options and Product details should appear

## Screenshots / Screen Captures (if appropriate)
![Zrzut ekranu 2020-10-24 o 12 57 23](https://user-images.githubusercontent.com/11998249/97080005-98872b00-15f8-11eb-95b8-79abe85076bb.png)
![Zrzut ekranu 2020-10-24 o 12 57 32](https://user-images.githubusercontent.com/11998249/97080008-9b821b80-15f8-11eb-9a61-81569c6677a1.png)
![Zrzut ekranu 2020-10-24 o 12 57 51](https://user-images.githubusercontent.com/11998249/97080010-9cb34880-15f8-11eb-8db5-5ec368f28114.png)
![Zrzut ekranu 2020-10-24 o 12 58 00](https://user-images.githubusercontent.com/11998249/97080013-9cb34880-15f8-11eb-8498-fe89886cf064.png)

## Checklist

* I have added tests to cover my changes, if necessary.
* I have added translations for new strings, if necessary.
* I have updated the documentation accordingly, if necessary.
